google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/facet/impl/AppEngineStandardUltimateWebIntegration.java

```diff
@@ -196,6 +240,12 @@ public void registerFrameworkInModel(
         .setFacet(AppEngineStandardFacetType.ID, appEngineStandardFacet);
   }
 
+  @NotNull
+  @Override
+  public String getDefaultAppEngineWebXmlPath() {
+    return "src/main/webapp/WEB-INF";
```

Comment:
didn't we say we were not going to use this strategy of assume this path? This is really only valid in special cases (maven directory structure).

Reply:
This is the path that would be used when then are no web resource directories.

Reply:
If we have no indication that this is the right path (i.e. no web roots, not web-inf) I'm not sure that its valid to assume that this is a good path (especially since it may not be a Maven project). @patflynn wdyt?

Reply:
I'm confused. My initial default path was "web/WEB-INF". I thought you wanted it changed to "src/main/webapp/WEB-INF".

Reply:
sorry - its possible Im contradicting myself. Here is the strategy as I see it (lmk if it makes sense to you):

1) if the web.xml exists use this directory
2) find the web roots - get the WEB-INF from it, and place it here
3) if there is no WEB-INF in the web root, create it and use this
4) we really don't have a good guess at this point so just fail and log (later in a different PR we can maybe do what @patflynn suggested about inspecting if its a Maven or Gradle project and putting it in that well known location)

Reply:
and just to point out - its arguable that step 1 should actually come after step 2 & 3. I'm open to either approach.